.github/actions/assign-issue-to-pr-author/AssignIssueToPullRequestAuthor.ps1

```diff
@@ -0,0 +1,28 @@
+param(
+    [string] $Body,
+    [string] $Assignee,
+    [string] $GitHubRepository
+)
+
+$Jira_Key = if ("$Body" -match '\[(.*?)\]')
```

Comment:
Shouldn't have this unconventional variable name and e.g. `$IssueQuery` below be caught by PS analysis, @BenedekFarkas @DAud-IcI?

Reply:
I see, thanks.

Reply:
How about this (added some extra details): https://github.com/Lombiq/PowerShell-Analyzers/issues/31